### PR TITLE
CORE-10691: Add topic for Avro schemas

### PR DESCRIPTION
**Pull request by fowlerrr**
Add a topic for Avro schemas.  This topic will be populated with avro schemas from previous versions in order to allow Avro to know how to deserialise older message versions.

Also fixed a bug to include fixed length avro messages which were missing from the generated list of supported messages .

### CODE REVIEW

**Review by simon-johnson-r3 — DISMISSED** [review]

File: data/avro-schema/build.gradle [hunk 1]
```diff
@@ -63,9 +63,14 @@ def generateOSGiPackageInfo = tasks.register("generateOSGiPackageInfo", Task) {
             if (!it.name.contains("package-info") && it.name.endsWith('.java')) {
                 def packageName = ((it.text =~ "package (.+);")[0][1])
                 packages << packageName
-                def classCapture = it.text.normalize() =~ "@org.apache.avro.specific.AvroGenerated\npublic class (\\w+) extends org.apache.avro.specific.SpecificRecordBase"
-                if (classCapture.any()) {
-                    classes << packageName + '.' + classCapture[0][1]
+                def normalizedText = it.text.normalize()
```
Comment (driessamyn) [inline data/avro-schema/build.gradle:66]: It's probably worth explaining in the PR description why this change is needed?

Comment (fowlerrr) [reply data/avro-schema/build.gradle:66]: Yep, done.

**Review by driessamyn — APPROVED** [review]
LGTM

**Review by mcgovc — APPROVED** [review]
LGTM
